### ISSUE TITLE
运行代码,网络是MAP-Net,结果IOU一直是0

### ISSUE THREAD

**Issue by Kurocuki** [lehaifeng/MAPNet#6]
Reading checkpoints...
Checkpoint not find 
train from scratch...
Total train image:3106
Total validate image:152
Total epoch:3
Batch size:4
Learning rate:0.001
Data Argument:
h_flip: True
v_flip: True
rotate: True
clip size: 450

除此之外,每一个epoch后,loss都是负的,没经过一次epoch 负号后面的值也越来越大....

**Comment by liwenjielongren** [comment]
请问，您解决这个问题了吗？我也是loss为负的

**Comment by liwenjielongren** [comment]
是因为没加载预训练模型吗

**Comment by 786942053** [comment]
> 是因为没有加载预训练模型吗？

请问你解决这个问题了吗
